Makefile.common

```diff
@@ -33,10 +35,15 @@ endif
 
 # Check that rustc is the right nightly - warn not error if wrong, sometimes useful to test others
 RUSTC_VERSION_STRING := rustc 1.16.0-nightly (83c2d9523 2017-01-24)
-ifneq ($(shell rustc --version),$(RUSTC_VERSION_STRING))
-    $(warning Unexpected rustc version. Expected $(RUSTC_VERSION_STRING) got $(shell rustc --version))
+ifeq ($(shell rustup -V &> /dev/null && echo success),success)
+  DUMMY := $(shell rustup install $(RUSTUP_TOOLCHAIN))
```

Comment:
I think this needs to fail / bail out here somehow if the install fails. Make doesn't care if the sub `$(shell` exits with an error code:

```
$ cat Makefile
FOO := /bin/false

all:
	echo hi
[-bash] Wed 22 Mar 16:21 [/tmp/m]
$ make
echo hi
hi
```